pkg/starboard/config.go

```diff
@@ -280,6 +327,15 @@ func (c ConfigData) GetKubeBenchImageRef() string {
 	return "docker.io/aquasec/kube-bench:0.4.0"
```

Comment:
could we get this from the default config settings? Would be nice to only have the default images defined in one place

Reply:
Certainly. I fixed it so the default is set once. When we read, but the property is not set we return error.